### PR TITLE
[Guide] get-started: new note of required subdomain for wrangler dev

### PR DESCRIPTION
**Pull request by nimeshnischal**

### CODE REVIEW

File: products/workers/src/content/get-started/guide.md [hunk 1]
```diff
@@ -244,6 +244,12 @@ Once you have filled in your `account_id`, you are ready to preview your code. R
 
 This command will build your project, run it locally, and return a URL for you to visit to preview the Worker.
 
+<Aside type="note">
+
+__Note:__ Before running `wrangler dev`, make sure you've created a subdomain. If not, follow the instructions in Step 1 to get a dev subdomain like `*.workers.dev`.
```
Comment (deadlypants1973) [inline products/workers/src/content/get-started/guide.md]:
```suggestion
Before running `wrangler dev`, make sure you have created a subdomain. If not, follow the instructions in Step 1 to get a dev subdomain like `*.workers.dev`.
```

**Comment by jkup** [comment]
Hey @nimeshnischal! Thank so much for the PR. Is there a particular problem you ran into here? I'm hesitant to merge it since you need to complete step 1 (registering an account) for most of the other steps to work. Let me know if there is a specific issue you ran into!

**Comment by WalshyDev** [comment]
Jon, I brought this up to Albert a few days ago. There have been many instances where users follow tutorials (for example Create React App - https://developers.cloudflare.com/workers/tutorials/deploy-a-react-app-with-create-react-app) and then cannot deploy due to a nondescript error which is resulted by this.

Example situation: https://discord.com/channels/595317990191398933/799437470004412476/899925648099905537

The error returned is simply `Error code 10007: workers.api.error.script_error_not_found`
No indication there is no subdomain. This is something that should be fixed both in Wrangler and mentioned in tutorials in my opinion.

**Comment by jkup** [comment]
@WalshyDev thank you for the additional context!

**Comment by KimJ15** [comment]
This PR was opened before the developer doc site was moved to Cloudflare Pages. There are many conflicting branches. I'm going to close it now, but feel free to open a new issue or ask any questions.

**Pull request closed without merging**